### ISSUE TITLE
Generalize to NxM image size

### ISSUE THREAD

**Issue by NicWayand** [antevis/CarND-Project5-Vehicle_Detection_and_Tracking#1]
Hi @antevis, thanks for a great write up. I am trying to apply your workflow to another image set with different dimensions and running into many errors related to hard coded image size/margin offsets. I am attempting to make it dynamic on the input image dimensions, but having trouble. Any suggestions you have would be appreciated. Thanks!

**Comment by antevis** [comment]
Hey Nic.

There are some pre-trained off-the-shelf solutions. I’d suggest using something from the Tensorflow detection model zoo: there are models for both bounding boxes and masks available. I've used it in the Capstone Project <https://github.com/antevis/CarND_Capstone>.

Ivan

**Comment by antevis** [comment]
This particular implementation isn't scalable to images of different dimensions **by design**.
